version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "partner_tag": "v5.2.1",
+  "debug_tag": "v0.0.1",
```

Comment:
Do we care that this is not camel case like the other two variables?

Reply:
This is used at build time only and only in this project. Should be safe to make it camel case